lib/webauthn/configuration.rb

```diff
@@ -20,9 +20,16 @@ def self.if_pss_supported(algorithm)
     attr_accessor :origin
     attr_accessor :rp_id
     attr_accessor :rp_name
+    attr_accessor :verify_attestation_statement
 
     def initialize
       @algorithms = DEFAULT_ALGORITHMS.dup
     end
+
+    def verify_attestation_statement?
+      return true if @verify_attestation_statement.nil?
```

Comment:
To make `true` the default value I think we could set it in the `#initialize` method, as we do for default algorithms. So we keep implementation consistency.

Reply:
ahh, 👍quite a bit better. thanks !